lint/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "jupiterp_linter"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
Question: Why are these included in the dependencies? They are necessary for `datagen` but I don't think they're necessary for this linter. If these were automatically included then that's alright

Reply:
Automatically included.